Formula/m/malcontent.rb

```diff
@@ -0,0 +1,20 @@
+class Malcontent < Formula
+  desc "Supply Chain Attack Detection, via context differential analysis and YARA"
+  homepage "https://github.com/chainguard-dev/malcontent"
+  url "https://github.com/chainguard-dev/malcontent/archive/refs/tags/v1.6.0.tar.gz"
+  sha256 "a8538888caee8d16fce62c398fc8eaa1af9aca225139afe9c8247c84202403db"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "pkgconf" => :build
+  depends_on "yara"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"mal"), "./cmd/mal"
+  end
+
+  test do
+    (testpath/"test.py").write("subprocess. python')")
```

Comment:
this looks better?
```suggestion
    (testpath/"test.py").write <<~PYTHON
        import subprocess
        subprocess.run(["echo", "execute external program"])
    PYTHON
```

Reply:
"execute external program" is the returned rule violation, so I'll use a different string here

Reply:
And to be clear - this doesn't have to be valid python - which is why I didn't need to `import subprocess` - do you still want a valid Python program as part of the test?